### PR TITLE
fix(cxl-ui): fix poor visibility of main category tag in cards

### PR DESCRIPTION
**Pull request by freudFlintstone**
https://app.clickup.com/t/86ayjuqbj

### CODE REVIEW

**Comment by lkraav** [comment]
Before / After screenshots directly in PR would help review things faster, on the go on mobile, etc.

**Comment by freudFlintstone** [comment]
@lkraav 
Before:
Screenshot from 2023-10-25 18-39-01
After:
Screenshot from 2023-10-25 18-39-15

**Review by pawelkmpt — APPROVED** [review]
Waiting for @heshfekry [reply](https://app.clickup.com/t/86ayjuqbj?comment=90140007509343) before merging

**Pull request merged**
